src/API/Status/PreviewCard.vala

```diff
@@ -121,6 +129,47 @@ public class Tuba.API.PreviewCard : Entity, Widgetizable {
 		}
 	}
 
+	#if CLAPPER_0_10
+		private bool enhancer_check (string scheme, string? t_host = null) {
+			if (!Clapper.WITH_ENHANCERS_LOADER) return false;
+			bool supported = false;
+
+			string? host = t_host;
+			if (host != null && host.has_prefix ("www.")) host = host.substring (4);
+
+			var proxies = Clapper.get_global_enhancer_proxies ();
+			for (var i = 0; i < proxies.get_n_proxies (); i++) {
+				var proxy = proxies.get_proxy (i);
+
```

Comment:
Missing check whether proxy enhancer is of "Extractable" type, see: [proxy.target_has_interface](https://rafostar.github.io/clapper/doc/clapper/method.EnhancerProxy.target_has_interface.html)